eventActions/bookmarkActions.js

```diff
@@ -0,0 +1,28 @@
+const config = require('../config.json');
+const Discord = require('discord.js');
+
+class bookmarkActions {
+	
+	static async bookmarkMessage(client, reaction) {
+		if (reaction._emoji.name === config.emotes.bookmark) {
+			const workingMessage = reaction.message;
+			workingMessage.guild.fetchMember(workingMessage.author.id).then(guildMember => {
+				const bookmarkEmbed = new Discord.RichEmbed()
+					.setColor('#0F9BF1')
+					.setTitle(`${config.emotes.cotwVow} Knights of Academia Bookmark ${config.emotes.cotwVow}`)
+					.setDescription('You asked to bookmark this post from the Knights of Academia server.')
+					.addField('Channel', workingMessage.channel)
+					.addField('Author', workingMessage.author);
+				const messageChunks = workingMessage.content.match(/[\s\S]{1,1024}/g);
+				
+				for (let chunk of messageChunks) {
```

Comment:
This could be const instead of let, since it's never changed

Reply:
I'm a little lost as to what you mean here, you mean change it to 
`for (const chunk of messageChunks) {` instead of the let? Both should work in theory, but I'm not the best judge of this so I want to double check before I make the change.

Reply:
Yep! Let values are for variables that'll be changed, since chunk never changes it can be a constant value.